CHANGELOG.md

```diff
@@ -23,26 +27,31 @@ clear(readTask);
 ```
 
 ## [2.0.0] - 2017-07-27
+
 ### Added
+
 - Adds a `CHANGELOG` to document changes moving forward.
 
 ### Changed
+
 - The `autobind` decorator is now in the `decorators` directory.
-    - **WARNING** any existing imports must be changed accordingly:
+  - **WARNING** any existing imports must be changed accordingly:
+
 ```ts
 // old way
 import autobind from '@shopify/javascript-utilities/autobind';
 
 // new way
 import {autobind} from '@shopify/javascript-utilities/decorators';
 ```
+
 - Updated documentation in `README.md` and `CONTRIBUTING.md`.
 
 ### Fixed
-- Multiple decorators can now be used simultaneously (ex. `autobind` with `debounce`).
 
+- Multiple decorators can now be used simultaneously (ex. `autobind` with `debounce`).
 
-[Unreleased]: https://github.com/shopify/javascript-utilities/compare/v2.2.0...HEAD
+[unreleased]: https://github.com/shopify/javascript-utilities/compare/v2.2.0...HEAD
```

Comment:
I think we have a bit backwards here. We'll need something closer to: 

```
[Unreleased]: https://github.com/shopify/javascript-utilities/compare/v2.2.1...HEAD
... releases in between 
[2.2.1]: https://github.com/shopify/javascript-utilities/compare/v2.2.0...v2.2.1
```

Reply:
Good catch 👍